packages/cubejs-testing/birdbox-fixtures/databricks-jdbc/cube.js

```diff
@@ -0,0 +1,9 @@
+// Cube.js configuration options: https://cube.dev/docs/config
+module.exports = {
+  queryRewrite: (query) => {
```

Comment:
Why do you need this? We do not use the Orders cube in the test suit.

Reply:
Just an accidental copypaste from Postgres